lib/venue.rb

```diff
@@ -1,2 +1,31 @@
 class Venue
+  attr_reader :name,
+              :capacity,
+              :patrons
+  def initialize(name, capacity)
+    @name     = name
+    @capacity = capacity
+    @patrons  = []
+  end
+
+  def add_patron(patron)
+    patrons << patron
+  end
+
+  def yell_at_patrons
```

Comment:
Nice job here Craig. Is there another enumerable that you could use so that you don't have to set an accumulator?